### ISSUE TITLE
Small, Wont fix? [Visual Requirement- NOVA-Query-MAS1.4.3] Luminosity contrast ratio of the text "MAX" is less than 4.5:1. 

### ISSUE THREAD

**Issue by khansahab15** [microsoft/monaco-editor#2311]
**GitHubTag:**#A11yMAS; #A11yTCS; #Win10; #DesktopWeb;#WCAG1.4.3; #A11ySev2; #Benchmark;#ChromiumEdge;#Chrome;#AI4D

**User Experience:**

If, all text elements does not have sufficient color contrast between the text in the foreground and background color behind it, then the users with low vision might get affected and does not able to read the available text available on the page properly.

**Repro steps:**

1. Open URL: https://novats.azurewebsites.net
2. Login via valid credentials
3. "Home" page will open.
4. Navigate to "Flows" menu item and select it.
5. Navigate to "New" button and select it.
6. "Info" screen will open.
7. Enter all valid data in "Info" section then navigate to "Input" section .
8. Enter all valid data in "Input" section then navigate to "Rules" section .
9. Enter all valid data in "Rules" section then navigate to "Query" section.
10. Navigate through all the controls appearing under "Query" (Theme, Preview, Run, Resample etc.) section.
11. Open CCA tool and check luminosity contrast ratio of text "MAX".
12. Verify, whether elements have sufficient color contrast or not.

**Actual Result:**

Luminosity contrast ratio of the text "MAX" is less than 4.5:1 i.e 3.1:1.

**Expected Result:**

Luminosity contrast ratio of the text "MAX" should be 4.5:1 or greater than 4.5:1.

Note: Same issue also exist with  MetricExample, Flowname and count text.

[Accessibility Insights](https://accessibilityinsights.io/)! - Identify accessibility bugs before check-in and make bug fixing faster and easier

**Test Environment:**
Version 1809 (OS Build 18314.1000)
Platform: Edge, Chrome
Screen Reader: Narrator , JAWS
![MAS1 4 3-Luminosity contrast ratio of highlighted text is less than 4 5 ratio 1](https://user-images.githubusercontent.com/25275622/105129563-58f2b100-5b0b-11eb-93f1-1ceef9ccfe7b.png)
